test/functional/sqlsrv/sqlsrv_encrypted_query_nosqltype.phpt

```diff
@@ -51,24 +90,28 @@ Test using sqlsrv_query for binding parameters with column encryption and a cust
 
     require_once('MsHelper.inc');
     $conn = AE\connect(array('ReturnDatesAsStrings'=>true));
-
-    if ($conn === false) {
-        echo "Failed to connect.\n";
-        printError();
-    } else {
+    if ($conn !== false) {
         echo "Connected successfully with ColumnEncryption enabled.\n\n";
     }
 
+    $AEQueryError = 'Must specify the SQL type for each parameter in a parameterized query when using sqlsrv_query in a column encryption enabled connection.';
+
     $tableName = createPatientsTable();
 
+    $numRows = 0;
+    
     $tsql = "INSERT INTO $tableName (SSN, FirstName, LastName, BirthDate) VALUES (?, ?, ?, ?)";
-    $inputs = array('748-68-0245', 'Jeannette', 'McDonald', '2002-11-28');
+    $patient = new Patient('748-68-0245', 'Jeannette', 'McDonald', '2002-11-28');
+    $inputs = array($patient->SSN, $patient->FirstName, $patient->LastName, $patient->BirthDate);
```

Comment:
since Patient is a simple type, you can cast it to an array `(array)$patient`

Reply:
I think I'd rather not change it now :)